src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -69,6 +73,23 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 });
 
+            services.AddHealthChecks()
+                .AddCheck(
+                    name: "self",
+                    check: () => HealthCheckResult.Healthy(),
+                    tags: new[] { HealthCheckTags.Live })
+                .AddSmtpHealthCheck(
+                    setup: (smtp) =>
```

Comment:
The brackets for the lambda parameter can be omitted here.

Reply:
👍